php/class-post-type.php

```diff
@@ -360,6 +362,7 @@ public function render_data_metabox( $post ) {
 			echo '<li>';
 			echo '<details open>';
 			echo '<summary><code>' . esc_html( $setting_id ) . '</code> ';
+			echo '<a href="#" id="' . esc_attr( $setting_id ) . '" data-text-restore="' . esc_attr( 'Restore setting', 'customize-snapshots' ) . '" class="snapshot-toggle-setting-removal remove">' . esc_html( 'Remove setting', 'customize-snapshots' ) . '</a>';
```

Comment:
Use translation function `esc_attr__` and `esc_html__`  instade of `esc_attr( 'Restore setting', 'customize-snapshots' )` and `esc_html( 'Remove setting', 'customize-snapshots' )`